spec/requests/atmosphere/api/endpoints_spec.rb

```diff
@@ -210,7 +210,7 @@
 
     context 'when unauthenticated' do
       it 'returns 401 Unauthorized error' do
-        post api("/endpoints")
+        post api("/endpoints"), new_request
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.